### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by GodKing9999** [arturzxc/myapex#17] [General]
Bro do you have a discord? I need help on how to compile the whole thing...

**Comment by LR-KING** [comment]
> Bro do you have a discord? I need help on how to compile the whole thing...

if you follow the readme it will have no issue on compile.and you can say what was wrong

**Reply by GodKing9999** [reply]
I tried bro and it is not working for me. Do you have Discord? Because you show me how to do it by streaming for me. Please

**Reply by LR-KING** [reply]
if you wanna solve problem just show us your error message instead of be an a**hole waiting someone's spoon feed

**Reply by GodKing9999** [reply]
I'll send the error message don't be mad mommy
